applications/asset_tracker/src/main.c

```diff
@@ -408,13 +411,23 @@ static void send_agps_request(struct k_work *work)
 		return;
 	}
 
+#if defined(CONFIG_NRF_CLOUD_CELL_POS)
+	LOG_INF("Sending cellular positioning request");
+
+	err = nrf_cloud_cell_pos_request(CELL_POS_TYPE_SINGLE, true);
+	if (err) {
+		LOG_ERR("Failed to request cell pos data, error: %d", err);
```

Comment:
```suggestion
		LOG_WRN("Failed to request cell pos data, error: %d", err);
```

Reply:
`LOG_ERR` should be reserved severe errors.

Reply:
Updated as advised